codex/validationconfig.nim

```diff
@@ -0,0 +1,36 @@
+import std/strformat
+import pkg/questionable
+import pkg/questionable/results
+
+type
+  ValidationGroups* = range[2..65535]
+  MaxSlots* = int
```

Comment:
🟡 Is there a particular reason this alias was used?

Reply:
Yes, I feel we still hesitate about the best type for `maxSlots`. This way it will be easier to change (otherwise you have to jump all over the places).